tools/tool-migration-utility/src/main/java/org/sagebionetworks/tool/migration/v3/SynapseClientFactoryImpl.java

```diff
@@ -42,7 +45,10 @@ public SynapseAdminClientImpl createNewDestinationClient() throws SynapseExcepti
 	 * @throws SynapseException 
 	 */
 	private static SynapseAdminClientImpl createNewConnection(SynapseConnectionInfo info) throws SynapseException{
-		SynapseAdminClientImpl synapse = new SynapseAdminClientImpl();
+		HttpClientProvider provider = new HttpClientProviderImpl();
+		provider.setGlobalConnectionTimeout(1000*60); 		// 	1 	min

```

Comment:
Why only one minute for the global timeout?   Some of the migration calls can take much longer (5+mins).